src/bg_palette_buffer.c

```diff
@@ -3,9 +3,9 @@
 #include "cpu.h"
 
 #define BG_PALETTE_BUFFER_SIZE 512
-#define BG_PALETTE_BUFFER_CHUNK_SIZE 16
+#define BG_PALETTE_ROW_SIZE 16
 
-static EWRAM_DATA bool8 sBGPaletteUsed[BG_PALETTE_BUFFER_SIZE / BG_PALETTE_BUFFER_CHUNK_SIZE] = {0};
+static EWRAM_DATA bool8 sBGPaletteRowDirty[BG_PALETTE_BUFFER_SIZE / BG_PALETTE_ROW_SIZE] = {0};
```

Comment:
What does `Dirty` mean here?

Reply:
It's a standard term for something being changed but not yet flushed or commited. Thus dirty caches, or a dirty git index.

Reply:
I see, thanks